test/e2e/framework/util.go

```diff
@@ -1641,14 +1642,17 @@ func LoadConfig() (*restclient.Config, error) {
 func LoadFederatedConfig() (*restclient.Config, error) {
 	c, err := restclientConfig(federatedKubeContext)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error creating federated client config: %v", err.Error())
 	}
 	cfg, err := clientcmd.NewDefaultClientConfig(*c, &clientcmd.ConfigOverrides{}).ClientConfig()
 	if cfg != nil {
 		//TODO(colhom): this is only here because https://github.com/kubernetes/kubernetes/issues/25422
 		cfg.NegotiatedSerializer = api.Codecs
 	}
-	return cfg, err
+	if err != nil {
+		return cfg, fmt.Errorf("error creating federated client config: %v", err.Error())
```

Comment:
OK.  I was just moving code from other places that already did this.  I will clean it up on the next PR.